staff/sergio-costa/cv/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV Sergio Costa</title>
+</head>
+<body>
+    <header>
+        <h1>CV Sergio Costa</h1>
+    </header>
+    <section>
+        <h3>Datos personales</h3>
+        <ul>
+            <li><b>Direccion:</b> C/Josep Irla 27</li>
```

Comment:
The W3C [discourages](https://developer.mozilla.org/es/docs/Web/HTML/Elemento/b#Notas) the use of `b` element.

Reply:
fixed ! 👍